### PR TITLE
Additional info form

### PR DESCRIPTION
**Pull request by nexussays**
https://app.asana.com/0/1186780595903796/1198517796726053

### CODE REVIEW

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      <p>
+         We are sharing your information with election administrators and our state partners who will follow up to help you be placed as a poll worker!&nbsp;
+        <strong>We just need a few more pieces of information from you to help with your application:</strong>
+      </p>
+
+      <hr />
+
+      <form onSubmit={submitForm} style={{ padding: "0" }}>
+
+         {data.state === "MI"
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:110]: could simplify to  by using logical AND `{data.state ==== "MI" && <label>`

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+                  type="text"
+                  required
+                  name="address1"
+               />
+            </label>
+            :
+            null
+         }
+
+         <Options type="radio" choices={["Yes", "No"]} inputName="action_workeday">
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:124]: spelling error: "action_workeday"

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+            Are you available to work on the Tuesday, November 3rd election? Hours may begin as early as 5:30am and go until at least an hour after polls close at 8pm. Can you commit to working the full day on Election Day?
+         </Options>
+
+         <Options type="radio" choices={["Yes", "No"]} inputName="action_waitlist">
+            If you are not selected to be a poll worker, would you be willing to serve on a backup waitlist?
+         </Options>
+
+         <Options type="select"
+            choices={[
+               "",
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:134]: Why have an empty string?

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+                     />{" "}{x}
+                  </label> )}
+            </div>
+         </div>
+      );
+
+/**
+ * Additional info be collected after users sign up
+ */
+const AdditionalInfoForm: FunctionalComponent<{ data: PtpFormData, onSubmit: () => void }> = ( { data, onSubmit } ) => {
```
Comment (Samcbehrens) [inline power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:41]: Are any of the fields in this form required ?

File: power-the-polls-form/src/data/States.ts [hunk 5]
```diff
@@ -11,10 +11,6 @@ export interface StateInfo {
     * For Maine. see: EmailApplicationForm.tsx
     */
    usePhoneInsteadOfEmailForFormFallback?: boolean;
-   /**
-    * Can't think of a better term
-    */
-   semiPartner?: boolean;
```
Comment (Samcbehrens) [inline power-the-polls-form/src/data/States.ts:17]: why are we removing semiPartner?

**Review by Samcbehrens — COMMENTED** [review]
Overall, looks good, just some small notes and questions.

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx (hunk 4 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:41]: They all are

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx (hunk 1 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:110]: I wanted to leave a placeholder for other states

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx (hunk 2 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:124]: Field names are from the original ticket, so even if it is wrong that's what the data looks like now

File: power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx (hunk 3 above)
Comment (nexussays) [reply power-the-polls-form/src/components/ptp-info-jurisdiction/AdditionalInfoForm.tsx:134]: Force the user to answer

File: power-the-polls-form/src/data/States.ts (hunk 5 above)
Comment (nexussays) [reply power-the-polls-form/src/data/States.ts:17]: It's no longer used anywhere

**Pull request merged**